src/locales/en/pages.yaml

```diff
@@ -225,13 +282,16 @@ honorary-members:
 
     * Lieke Arts
     * Leo Rijkaart
-    * Daan Admiraal
+    * Daan Admiraal (†, [In memoriam](/daan))
```

Comment:
Raleway does not have a [dagger](https://en.wikipedia.org/wiki/Dagger_(mark)) character (†). On the website it displays as a [cross box drawing character](https://en.wikipedia.org/wiki/Box-drawing_character#Box_Drawing) which looks a bit weird (┼).

You could put a span around this dagger which forces the font to be the default sans-serif font:

```html
    <span style="font-family: sans-serif">†</span> ...
```

Though I am not sure if YAML likes HTML in its syntax.

Reply:
Good idea! Yes, html syntax works if you create a Markdown parent element with ```options={{ html: true }}```